.azure/azure-pipelines.ci.yml

```diff
@@ -659,6 +659,17 @@ stages:
       platform: windows
       tls: schannel
       logProfile: Full.Light
+  - template: ./templates/run-bvt.yml
```

Comment:
How can I get this to work?

Reply:
I am getting this error: `Stage test_bvt job bvt_windows_x64_schannel_false_false_windows_2022_Xdp appears more than once. Job names must be unique within a stage.`

Reply:
I think there's an arg like extra name you need to set to something unique

Reply:
Ah, I see. The parameters form the job name in run-bvt.yml. However, the job name doesn't include extraName. I will remove this for name and handle it in a separate PR after getting the github actions stable.